spec/libraries/sign_in_with_callback_spec.rb

```diff
@@ -20,7 +20,8 @@ def after_callback
   let(:instance) { Instance.default }
   with_tenant(:instance) do
     let(:user) { self.class::TestUser.new }
-    subject { controller.sign_in(:user, user, bypass: true) }
+    # Set run_callbacks to false to skip Warden callbacks
+    subject { controller.sign_in(:user, user, run_callbacks: false) }
```

Comment:
why need to skip Warden callbacks?

Reply:
Same effect as `bypass: true`, which has been deprecated.

Need to skip because the tests here are using a dummy user class.

Reply:
OK 👍